content/soarian/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -0,0 +1,94 @@
+---
+title: DiagnosticReport | Soarian DSTU 2 API
+---
+ 
+# DiagnosticReport
+
+* TOC
+{:toc}
+
+## Overview
+
+For the DiagnosticReport resource query, the API returns test names and result values for test types that the Soarian client has configured as appropriate for the API.  The ONC (Office of the National Coordinator) CCDS (common clinical data set) definition is defined to include laboratory tests and values however clients may configure additional qualifying test types such as radiology, cardiology, etc.  The configuration of qualifying test types is a single configuration that applies to both the API and the Consolidated Clinical Document Architecture (C-CDA) document available to the patients. 
+
+The API returns the test names and values in reverse chronological order by result date and, within that, the sequence that is configured in the Soarian flowsheet that defines the qualified tests.  Qualified results of status Final, Corrected, or Cannot Obtain are returned.  Result values of numeric or text are returned; image only results, including those encapsulated in non-text formats such as pdf, are not returned.  Since each result is individually dated and the value is unambiguously relevant for that date, this resource supports date-based queries.
+
+## Terminology Bindings      
+
+
+<%= terminology_table(:soarian_diagnostic_report, :dstu2) %>
+
+## Extensions
+The content of these extensions must be visible to the user, and recorded with any storage or transmit activity along with the query/response date/time, as it represents the best representation of what clinicians recorded for the patient at the time of query.  
+
+
+| Name | Condition | Description |
+| --- | --- | --- |
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|No qualifying entries exist|**No relevant clinical data exists**|
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Qualifying entries exist|The following labels and values from the list will be returned in HTML, if valued: **Test/Procedure Name, Test Value, Value Unit, Reference Range, micro (culture) results**|
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Qualifying entries exist|Informational message that must be shown to the user: **Information provided is from YYYY-MM-DD through YYYY-MM-DD inclusive.**|
+
+**CAUTION:** The detail that is in the narrative may not be fully expressed in the structured portion of the FHIR response: The https://www.hl7.org/fhir/narrative-definitions.html#Narrative.status entry in the narrative is valued as "Additional" indicating that more information is provided in the narrative than is provided in the structure. Therefore, if the Application is using the structured entries for any purpose, care must be taken to ensure the clinical content/context of the narrative is preserved.  
```

Comment:
This link needs to specifically reference dstu 2 on the HL7 site. This doc is now pointing at STU 3 (and will continue to point to "latest")